src/main/java/racingcar/model/Cars.java

```diff
@@ -0,0 +1,51 @@
+package racingcar.model;
+
+import java.util.List;
+import racingcar.dto.RaceResultDTO;
+import racingcar.exception.ErrorMessage;
+import racingcar.exception.InvalidInputException;
+
+public class Cars {
+
+    private final List<Car> cars;
+
+    public Cars(CarNames carNames) {
+        cars = carNames.getCarNames().stream()
+                .map(Car::createWithName)
+                .toList();
+    }
+
+    public static Cars createWithNames(CarNames carNames) {
+        return new Cars(carNames);
+    }
+
+    public List<RaceResultDTO> move() {
+        return cars.stream().map(Car::move).toList();
+    }
+
+    public List<String> getWinner() {
+        return cars.stream()
+                .filter((car) -> isWinner(car, getFarthestLocation()))
+                .map(Car::getName)
+                .toList();
+    }
+
+    private int getFarthestLocation() {
+        return cars.stream()
+                .mapToInt(Car::getLocation)
+                .max()
+                .orElseThrow(() -> InvalidInputException.with(ErrorMessage.NO_CARS));
+    }
+
+    private static boolean isWinner(Car car, int farthestLocation) {
+        if (car.getLocation() == farthestLocation) {
+            return true;
+        }
+        return false;
+    }
```

Comment:
```suggestion
    private static boolean isWinner(Car car, int farthestLocation) {
            return car.getLocation() == farthestLocation;
    }
```
이렇게 작성할 수 있을 것 같습니다~!

Reply:
그러네요... 이걸 생각 못하다니.. 감사합니다